src/main.cc

```diff
@@ -85,101 +92,157 @@ Global Options:
 
 For more documentation, visit https://github.com/sourcemeta/jsonschema
 )EOF"};
+// Initialize color usage
+bool use_colors = true;
 
 auto jsonschema_main(const std::string &program, const std::string &command,
-                     const std::span<const std::string> &arguments) -> int {
-  if (command == "fmt") {
+                     const std::span<const std::string> &arguments) -> int
```

Comment:
All of these changes seem unrelated? They will also break the build, and we check formatting against `ClangFormat`

Reply:
If you run `make` on your system that should run `ClangFormat` as per the project's configuration and always get the formatting right

Reply:
Yeah i am this one only Thankyou you are great help.😃